installer/scripts/omsadmin.sh

```diff
@@ -427,6 +433,7 @@ onboard()
     
     if [ $error -ne 0 ]; then
         log_error "Error during the onboarding request. Check the correctness of the workspace ID and shared key or run omsadmin.sh with '-v'"
+        rm "$FILE_CRT" "$FILE_KEY" > /dev/null 2>&1 || true
```

Comment:
This is the onboard method. Although the cert/key is good, but it's not registered to the OMS serve if there is a server outage, the client should not reuse the cert/key.